File: pizza-api/pizza-data/models/PizzaOrderDetail.cs
using System.Text.Json.Serialization;

namespace pizza_data.models
{
    public class PizzaOrderDetail
    {
        public PizzaOrderDetail() { }
        public PizzaOrderDetail(PizzaOrder order, Topping topping)
        {
            this.PizzaOrderId = order.Id; this.ToppingId = topping.Id;
        }

        public int Id { get; internal set; }
        [JsonIgnore]
        public int PizzaOrderId { get; set; }
        public virtual PizzaOrder PizzaOrder { get; set; }
        [JsonIgnore]
        public int ToppingId { get; set; }
        public virtual Topping Topping { get; set; }
    }
}

File: pizza-sequelize/models/pizzaorderdetail.js
'use strict';
module.exports = (sequelize, DataTypes) => {
  const pizzaorderdetail = sequelize.define('pizzaorderdetail', {
    /*pizzaorderid: {
      type: DataTypes.INTEGER,
      references: {
         model: 'pizzaorder',
         key: 'id',
      }
    },
    toppingid: {
      type: DataTypes.INTEGER,
      references: {
         model: 'topping',
         key: 'id',
      }
    }*/
  }, {});
  pizzaorderdetail.associate = models => {
    // associations can be defined here
    pizzaorderdetail.belongsTo(models.pizzaorder, {
      foreignKey: {
        allowNull: false
      }
    });
    pizzaorderdetail.belongsTo(models.topping, {
      foreignKey: {
        allowNull: false
      }
    });
  };
  return pizzaorderdetail;
};